google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITTransferManagerTest.java

```diff
@@ -216,6 +220,41 @@ public void downloadBlobsAllowChunked() throws Exception {
     }
   }
 
+  @Test
+  public void uploadNonexistentBucket() throws Exception {
+    TransferManagerConfig config =
+        TransferManagerConfigTestingInstances.defaults(storage.getOptions()).toBuilder().build();
+    String bucketName = "this-bucket-does-not-exist";
```

Comment:
nit: someone could create this bucket. To reduce fragility, instead modify the name of a bucket we manage in the scope of our test suite https://github.com/googleapis/java-storage/blob/bfe0415882e241c60147018abeacda3daabd6653/google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITBucketAclTest.java#L87

(This also help act as a "bucket name typo'd" type case)